templates/master/00-master/azure/files/opt-libexec-openshift-azure-routes-sh.yaml

```diff
@@ -80,35 +80,40 @@ contents:
 
     # set the chain, ensure entry rules, ensure ESTABLISHED rule
     initialize() {
-        ensure_chain4 nat "${CHAIN_NAME}-local"
```

Comment:
why are we removing the `-local` suffix?

Reply:
because the previous chain was referring only to local clients, that is no longer the case